### PR TITLE
Feature/cleanUp

### LINKED ISSUE #15: About Page?
NavLink in Header that takes User to Page of info about site.
Could be Legend, description of site and devs, etc.

### LINKED ISSUE #43: About page bug
- [ ] Test
- [x] Fix Bug
When you navigate to the About page, this error appears in the console:

`Warning: validateDOMNesting(...): Text nodes cannot appear as a child of <thead>.
    in thead (at About.tsx:76)
    in table (at About.tsx:75)
    in div (at About.tsx:69)
    in section (at About.tsx:23)
    in About (at App.tsx:44)
    in Route (at App.tsx:41)
    in main (at App.tsx:34)
    in div (at App.tsx:30)
    in App (at src/index.tsx:8)
    in Router (created by BrowserRouter)
    in BrowserRouter (at src/index.tsx:8)`

### PR DESCRIPTION
**Pull request by leighlars**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
This PR addresses some areas of clean up.
The `About` page renders all icons in legend and is completely tested. 
The `Location` page renders AQI data in the AQI box with dynamic color matching 
Methods `temp`, `aqIndex`, and `uvIndex` have all been moved to `conversions` and then imported into `Card` and `Location`. The styling may be off in `Card`, and I'll address that when we can load our site. 

close #43, #67, #66, #15  
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Other, Please explain:
Not really anything new, just stuff polished. 
## Has This Been Tested?
- [X] No
- [X] Yes 
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
`About` tests are completed and passed.

# Checklist:
- [X] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/About/About.test.tsx [hunk 1]
```diff
@@ -15,9 +15,40 @@ describe('App', () => {
 		expect(aqTable).toBeInTheDocument()
 	})
 
-	it('should display a summary of site', () => {
-		const summary = screen.getByText(/whether you\'re planning a trip or/i)
+	it('should display a summary of site with links', () => {
```
Comment (ErinUntermeyer) [inline src/About/About.test.tsx:18]: Yay testing!!

File: src/Card/Card.tsx [hunk 2]
```diff
@@ -1,21 +1,8 @@
...
-// import rain from '../assets/09d.png'
-// import daySunnyStorm from '../assets/10d.png'
-// import sunnyStorms from '../assets/10n.png'
-// import thunderstorm from '../assets/11d.png'
-// import thunderRain from '../assets/11n.png'
-// import snow from '../assets/13d.png'
-// import mist from '../assets/50d.png'
 
-import { weatherIcon } from '../helpers/conversions'
+import { weatherIcon, temp, aqIndex, uvIndex } from '../helpers/conversions'
```
Comment (ErinUntermeyer) [inline src/Card/Card.tsx:5]: No more clutter! Love it.

File: src/helpers/dataFilter.tsx [hunk 3]
```diff
@@ -1,33 +1,28 @@
...
-	data.temp = +((weather.current.temp - 273.15) * 1.8 + 32).toFixed(0)
+	data.temp = +weather.current.temp 
 	data.uvi = +(Math.round(weather.current.uvi)).toFixed(0)
 	data.icon = weather.current.weather[0].icon
 	// if (aq.length !== 0) {
-	// 	data.aqi = aq[0].AQI
-	// 	data.aqiCat = aq[0].Category.Name
-	// console.log(data)
-	// }
+		// 	data.aqi = aq[0].AQI
```
Comment (ErinUntermeyer) [inline src/helpers/dataFilter.tsx:22]: Can we remove the commented out code here since we are not displaying air quality data on home cards?

**Comment by ErinUntermeyer** [comment]
Thanks for cleaning up Leigh!

File: src/About/About.test.tsx [hunk 4]
```diff
@@ -15,9 +15,40 @@ describe('App', () => {
...
+	})
+
+	it('should display an icon legend', () => {
+		const iconHeader = screen.getByText('Icon Legend')
+		expect(iconHeader).toBeInTheDocument()
+	})
+
+	it('should display credits box with links to sources and devs', () => {
+		const erin = screen.getByRole('link', {name: 'Erin Untermeyer'})
+		const josh = screen.getByRole("link", { name: "Josh Sevy" })
```
Comment (JoshSevy) [inline src/About/About.test.tsx:42]: We famous

**Review by ErinUntermeyer — APPROVED** [review]

**Review by JoshSevy — COMMENTED** [review]
Looks great. Great work! Crushing it.

**Review by JoshSevy — APPROVED** [review]

**Pull request merged**